src/app/docs-pages/popup-connected-doc/popup-connected-doc.component.html

```diff
@@ -15,12 +19,21 @@ <h1 class="doc-title">Popup connected component
         https://material.angular.io/cdk/overlay/api
       </a>
     </p>
-    <p class="doc-info"> 
-      <b>
-        Typical use: Displaying some kind of information where you want the popup to be connected to an element. 
-      </b>
-    </p>
-  
+
+    <div class="doc-info-use">
+      <h2><i class="fal fa-check-circle doc-info-use-icons icon-green"></i> <b>When to use:</b></h2>
+      <p>
+        Dialogs are used now that the user's attention must be directed to important information. 
+        The connected popup should be used when you want to display information that is connected to an element. 
```

Comment:
Bør ha med at popups bør ha en action?

Reply:
Typ som i "ok, jag fattar" eller "gör X eller Y". 

Vidare att en popover/connected popup ersätter en tooltip om mängden information är mer än feks två meningar, eller tillför extra information om ett objekt.

Reply:
Tror vi har en forskjellig forståelse av en popup. Jeg ser på de hovedsakelig som handlinger som skal utføres, som og sette et filter, men jeg tolker det at du ser på de som informasjonspopups, en forstørret tooltip. Akkurat nå brudkes de mest som handlinger, i mdm/drop ihvertfall

Reply:
Yes, din tolkning stämmer bra. Men då är det säkert fine. Tycker att det är aningen svårläst att sätta sig in i dokumentationen bara via en pull request utan att sitta på branchen och se hur det ser ut.

Reply:
"The connected popup should be used when you want to display information or action(s) that's connected to an element."